jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -1297,10 +1307,23 @@ static void initSecurityManager(ClassLoader applicationClassLoader) {
 				throw new Error(Msg.getString("K0631", e.toString()), e); //$NON-NLS-1$
 			}
 		}
+		/*[ENDIF] JAVA_SPEC_VERSION >= 24 */
 	}
+	/*[IF JAVA_SPEC_VERSION >= 24]*/
+	initPhase3(throwErrorOnInit);
+	/*[ENDIF] JAVA_SPEC_VERSION >= 24 */
 }
 /*[ENDIF] JAVA_SPEC_VERSION >= 9 */
 
+/*[IF JAVA_SPEC_VERSION >= 24]*/
+private static void initPhase3(boolean throwError) {
+	if (throwError) {
+		/*[MSG "K0B04", "A command line option has attempted to allow or enable the Security Manager. Enabling a Security Manager is not supported."]*/
+		throw new Error(Msg.getString("K0B04")); //$NON-NLS-1$
+	}
+}
+/*[ENDIF] JAVA_SPEC_VERSION >= 24 */
```

Comment:
Would it cause any problems to simply rename `initSecurityManager` to `initPhase3`?

Reply:
That should be fine. I renamed the method and added a description to clarify what the method does.